README.md

```diff
@@ -113,148 +111,10 @@ netlify deploy # enter ./build as the deploy path
 
 You'll then simply follow the command prompt to log into Netlify and deploy your site. This can be completed in just a few minutes.
 
-## Understanding the Reference UI Code
```

Comment:
Just for my edification - this documentation section moved to Search UI, right?

Reply:
@constancecchen Yes.